core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1892,7 +1892,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
     }
     val callSite = getCallSite
     val cleanedFunc = clean(func)
-    logInfo("Starting job: " + callSite.shortForm)
+    logDebug("Starting job: " + callSite.shortForm)
```

Comment:
This is not job server's job. It is internal job for Spark. There will be too many of them. Secondly, what would one do with this job id in the log.